assets/css/global-custom.css

```diff
@@ -543,6 +543,11 @@ body{
     padding: .5rem .75rem;
 }
 
+.round-top
```

Comment:
let's rename this to `rounded-top`

Reply:
Tried to rename the class name as `rounded-top`. But it isn't working. Therefore renamed it to `round-top`.

Reply:
oh alright then it's fine